### PR TITLE
Allow passign in value to ocRadio

### PR DESCRIPTION
**Pull request by butonic**
allow passing in an actual value
needed for https://github.com/owncloud/phoenix/pull/913

### CODE REVIEW

File: src/patterns/OCSharingStatus.vue [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+<script>
+/**
+ * The sharing sidebar
+ *
+ * ##TODO:
+ *
+ * - [ ] fix selecting roles
+ * - [ ] emit save event
+ * - [ ] use mixin to format file size
```
Comment (DeepDiver1975) [inline src/patterns/OCSharingStatus.vue]: job of phoenix

File: src/patterns/OCSharingStatus.vue [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ * - [ ] use mixin to format mtime
```
Comment (DeepDiver1975) [inline src/patterns/OCSharingStatus.vue]: job of phoenix

File: src/patterns/OCSharingStatus.vue [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    },
+    /**
+     * Sharing roles
+     **/
+    roles: {
+      type: Object,
+      required: false,
+      default() {
+        return {
+          viewer: { name: "Viewer", description: "Download and preview" },
```
Comment (DeepDiver1975) [inline src/patterns/OCSharingStatus.vue]: this should all go into the example code below

File: src/elements/OcRadio.vue [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
   <label class="radio-container">
     <span class="label">{{ label }}</span>
-    <input type="radio" v-model="model" name="radio" :aria-label="label" @change="onChange" />
+    <input
+      type="radio"
+      v-model="model"
+      :value="value"
+      name="radio"
+      :aria-label="label"
+      @change="onChange"
```
Comment (DeepDiver1975) [inline src/elements/OcRadio.vue]: please use naming convention - see https://vuejs.org/v2/style-guide/#Private-property-names-essential

File: src/patterns/OCCollaborationStatus.vue [hunk 5]
```diff
@@ -0,0 +1,199 @@
+<template>
+  <div class="oc-collaboration-status">
```
Comment (DeepDiver1975) [inline src/patterns/OCCollaborationStatus.vue]: The component itself is find - but to me this is something to be implemented in phoenix.

**Comment by butonic** [comment]
moved collaboration component to https://github.com/owncloud/phoenix/pull/913
all that is left is the chagnes to ocRadio, naming guidelines now followed

**Pull request merged**